sdk/keyvault/azkeys/client.go

```diff
@@ -98,8 +98,11 @@ func (c *CreateKeyOptions) toKeyCreateParameters(keyType KeyType) generated.KeyC
 	}
 
 	var ops []*generated.JSONWebKeyOperation
-	for _, o := range c.KeyOps {
-		ops = append(ops, (*generated.JSONWebKeyOperation)(o))
+	if c.KeyOps != nil {
+		ops = make([]*generated.JSONWebKeyOperation, len(c.KeyOps))
```

Comment:
The name the other SDKs use is just `KeyOperations`.

Reply:
Updated